docs/platforms/powershell/data-management/data-collected.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
```

Comment:
Do we need to copy paste the whole thing? 
We could just say "it's the same as .NET, go see on that page".